apps/console/src/features/identity-providers/components/wizards/authenticator-create-wizard-factory.tsx

```diff
@@ -452,6 +453,24 @@ export const AuthenticatorCreateWizardFactory: FunctionComponent<AuthenticatorCr
                     />
                 )
                 : null;
+        case IdentityProviderManagementConstants.IDP_TEMPLATE_IDS.IPROOV:
+            return (showWizard && !isEmpty(selectedTemplateWithUniqueName))
+                ? (
+                    <IproovAuthenticationProviderCreateWizard
```

Comment:
You don't need a seperate wizard for Iproov, since we can use the dynamic wizard.